Back up GitHub content

Just for belt-and-suspenders purposes, we should back up all of the useful content on GitHub that is not in the core Git repository -- issues, pull requests comments, release binaries, etc.

I'm sure there's a tool to do this. Backups could be as simple as dumping it all into a Git repository that is pushed to another hosting service.

This does not appear to be working! This may be partially due to API limits, which may not be an issue if we didn't download the HTML. It fails with logs ending with
```
Done downloading JSON
repo: GothenburgBitFactory/taskwarrior
entries: 3940
Download issues and pull requests HTML sequentially
Error: Process completed with exit code 1.
```

--- a/.github/workflows/backup.yml
+++ b/.github/workflows/backup.yml
@@ -35,10 +35,10 @@ jobs:
 
       - name: "Configure backup-github-repo"
         run: |
-          echo "{\"token\":\"${GH_TOKEN}\"}" > .backup-github-repo
+          echo "{\"token\":\"${GITHUB_TOKEN}\"}" > .backup-github-repo
           echo 	.backup-github-repo >> .git/info/exclude
         env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: "Run backup-github-repo"
         run: backup-github-repo --no-html

